.github/workflows/release.yml

```diff
@@ -36,7 +36,7 @@ jobs:
         run: make build-skore-ui
 
       - name: Override VERSION.txt with tag
-        run: echo "${GITHUB_REF_NAME}" > VERSION.txt
+        run: echo "${GITHUB_REF_NAME}" > skore/VERSION.txt
```

Comment:
Isn't it a good time to remove this generated file from the sources N

Reply:
i will do it, but in another PR.

Reply:
Perfect then.